src/graph/test/SchemaTest.cpp

```diff
@@ -135,40 +156,149 @@ TEST_F(SchemaTest, metaCommunication) {
         ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
         std::string createSpaceStr = "CREATE SPACE default_space ("
                                      "partition_num = 9, "
-                                     "replica_factor = 1)";
```

Comment:
the charset and collate should matching ? Or support a check?

Reply:
Yeah，there is  a function    `charsetAndCollateMatch`.